docs/tutorials/mastering.md

```diff
@@ -1,60 +1,626 @@
+        - vlan203
+    ran-br:
+      addresses:
+        - 10.204.0.14/24
+      routes:
+        - to: 10.202.0.0/24
+          via: 10.204.0.1
+      interfaces:
+        - vlan204
+
+  vlans:
+    vlan201:
+      id: 201
+      link: enp4s0
+    vlan202:
+      id: 202
+      link: enp4s0
+    vlan203:
+      id: 203
+      link: enp4s0
+    vlan204:
+      id: 204
+      link: enp4s0
+
+  version: 2
+EOF
+```
+
+```console
+sudo netplan apply
+```
+
+### Install and Configure LXD
+
+Install LXD:
+```console
+sudo snap install lxd
+```
+
+While initializing the LXD, please pay attention to set the size of new loop device and local network bridge. 
+
+- Loop device size should be enough to launch all VMs
+- New lxd bridge should not be created.
+
+Use default settings for other configurations.
+
+```console
+$ lxd init
+Would you like to use LXD clustering? (yes/no) [default=no]: 
+Do you want to configure a new storage pool? (yes/no) [default=yes]: 
+Name of the new storage pool [default=default]: 
+Name of the storage backend to use (dir, lvm, zfs, btrfs, ceph) [default=zfs]: 
+Create a new ZFS pool? (yes/no) [default=yes]: 
+Would you like to use an existing empty block device (e.g. a disk or partition)? (yes/no) [default=no]: 
+Size in GiB of the new loop device (1GiB minimum) [default=30GiB]: 150GiB
+Would you like to connect to a MAAS server? (yes/no) [default=no]: 
+Would you like to create a new local network bridge? (yes/no) [default=yes]: no
+What IPv4 address should be used? (CIDR subnet notation, “auto” or “none”) [default=auto]: 
+What IPv6 address should be used? (CIDR subnet notation, “auto” or “none”) [default=auto]: 
+Would you like the LXD server to be available over the network? (yes/no) [default=no]: 
+Would you like stale cached images to be updated automatically? (yes/no) [default=yes]: 
+Would you like a YAML "lxd init" preseed to be printed? (yes/no) [default=no]: 
+```
+
+### Install and configure Multipass
+
+Install Multipass and set LXD as local driver:
+
+```console
+sudo snap install multipass
+sudo multipass set local.driver=lxd
+```
+Wait a few seconds if you get the output: `set failed: cannot connect to the multipass socket` and retry to set local driver again.
+
+Check if local driver is set to lxd properly.
+```console
+$ sudo multipass get local.driver
+lxd
+```
+
+Connect multipass to LXD.
+```console
+$ sudo snap connect multipass:lxd lxd
+```
+
+Check multipass connections and make sure that `multipass:lxd` appears under Plug:
+```console
+$ sudo snap connections multipass
+Interface          Plug                         Slot                Notes
+lxd                multipass:lxd                lxd:lxd             -
+```
+
+Set Multipass passphrase and authenticate:
+```console
+$ sudo multipass set local.passphrase
+Please enter passphrase: <set_your_passphrase>
+Please re-enter passphrase: <set_your_passphrase> 
+
+$ multipass authenticate
+Please enter passphrase: <your_passphrase>
+```
```

Comment:
Are those necessary for a tutorial?

Reply:
yes, they are required. Otherwise we will get following error.
```
launch failed: The client is not authenticated with the Multipass service.
Please use 'multipass authenticate' before proceeding.
```